test/tools/fuzzing/main.cc

```diff
@@ -0,0 +1,18 @@
+#include "options_boost_po.h"
+
+#include "vw.h"
+#include "vw_exception.h"
+#include <memory>
+
+int main(int argc, char** argv)
+{
+  std::unique_ptr<VW::config::options_i> ptr(new VW::config::options_boost_po(argc, argv));
+  try {
+    vw* all = VW::initialize(*ptr);
```

Comment:
You need to call finish otherwise this is a leak

Reply:
Finish is slow, and I don't particularly care if there's a leak. The process will get scrapped and all the memory will get cleaned up regardless of what happens. This test is literally just testing the initialization step

Reply:
You can call delete on all, which should be faster than finish?

Reply:
There are a lot of crashes we won't catch with this test, notably anything dealing with the learners. I say we set up another test that does a full end-to-end workflow. It'll be MUCH slower, but there's not much we can do about that